src/ImageClassifier/index.js

```diff
@@ -263,11 +263,9 @@ const imageClassifier = (modelName, videoOrOptionsOrCallback, optionsOrCallback,
   let options = {};
   let callback = cb;
 
-  let model = modelName;
+  const model = modelName;
```

Comment:
blocking change:
* We'll have to update our examples or how how constructor is handling how our default models are referenced -- https://github.com/ml5js/ml5-library/pull/1339/files#diff-a06592fcb56a6560078a1c33d5a3bcd5909c2bf732c347ed1c0cd64096c81a1bR27-R65 
* You can see how we input the model name like so: https://github.com/ml5js/ml5-library/blob/main/examples/p5js/ImageClassification/ImageClassification_DoodleNet_Canvas/sketch.js#L24

Reply:
Oh my gosh…I read the `if (model.indexOf('http') === -1)` backwards and I was thinking that we are changing the case if IS a URL, but actually we’re changing it if it’s NOT a URL!

Probably it makes more sense to handle the lower-casing in the `switch` statement: `switch (model.toLowerCase())`.  That is, at the point where we match the model names rather than at the point where we provide the model names.  That way we don’t need to worry about whether it’s a URL or not.  Like what if they provide a relative file path but it’s case-sensitive?  That wouldn’t be handled properly right now because it doesn’t match `http`.

So this PR is not ready as-is because I completely misunderstood it, but I will take another look and see if I can make this better.